doc/usage/theming.rst

```diff
@@ -155,6 +155,21 @@ These themes are:
     previous/next page using the keyboard's left and right arrows.  Defaults to
     ``False``.
 
+  .. versionadded:: 3.1
```

Comment:
This breaks the bullet list. It causes the document hard to read.  Could you move this under each list item?

Reply:
Done: 3baa6e09e2c488a29c2d6d18172aa47422dc4f1c